index.bs

```diff
@@ -12031,7 +12031,22 @@ class Vec3 {
 
 <h2 id="changelog">
 Change Log</h2>
-<h3 id="changestart">
+<h3 id="changestart1">
+Since Working Draft of 19 June 2018</h3>
+* Update implementation-report.html
+* Widen range for detune values
+* PannerNode constructor throws errors
+* Rephrase algorithm for setting buffer and curve
+* Refine startRendering algorithm.
+* Spec more precisely events overlapping with SetValueCurveAtTime
+* Add implementation report to gh-pages
+* Honor the given value in `outputChannelCount`
+* Add images for convolution doc
```

Comment:
The convolution doc isn't part of the spec, so I'd leave this out.(If by convolution doc you mean convolution.html)

Reply:
Right, this is in our repo but doesn't get copied over when the spec is published

Reply:
yes, this one crept through my filter.